include/mbedtls/bn_mul.h

```diff
@@ -630,6 +630,23 @@
            "r6", "r7", "r8", "r9", "cc"         \
          );
 
+#elif defined(__ARM_FEATURE_DSP)
```

Comment:
Should this be `#if defined(__ARM_FEATURE_DSP) && (__ARM_FEATURE_DSP == 1)`?

Reply:
You are indeed correct. I have check CMSIS code and the check is done that way.